### DISCUSSION TITLE
Bash characters need escaping

### DISCUSSION THREAD

**Discussion by RamiAwar** [knqyf263/pet#276] [Ideas]
This is not a pet issue per-say, but pet can help alleviate it a little.

`echo "Hello world!"` will fail because the exclamation mark is a special character. 

Idk if we should do this, only reason against is what if people are actually using the exclamation mark in their snippets. What do you think @knqyf263 ? 

![pet-search-bash-unescaped](https://github.com/knqyf263/pet/assets/8523191/ce6461d4-a5c5-4dfe-97f4-0c3d611d478f)

**Comment by knqyf263** [comment]
I think `pet` should work as well as a shell. In other words, `echo "Hello world!"` does not work on bash and echo `"Hello world\!"` works on bash, then `pet` should do the same. Users should escape it themselves, and `pet` doesn't need to escape it on our end. Does it answer your question?

**Reply by RamiAwar** [reply]
Yeah that's what I was thinking too. I think that's a good philosophy. Thanks!
